src/common/launchArgs.ts

```diff
@@ -12,8 +12,12 @@ export interface IRunOptions extends ILaunchArgs {
  * Defines the supported launch arguments.
  * Add more arguments here as needed.
  */
-interface ILaunchArgs {
+export interface ILaunchArgs {
     platform?: string;
     target?: string;
     debugAdapterPort?: number;
+    extensionServerPort?: number;
+    /* internal properties */
+    internalDebuggerPort?: any;
+    args?: string[];
```

Comment:
This is currently conflating 2 different launch args: The original ILaunchArgs here was arguments-as-interpreted-by-debugee, and was constructed based on command line arguments. The new interpretation adds in arguments-as-specified-in-launch.json which is slightly different.

For example the `args` property is the nodejs debugger args property which we use to pass arguments to the debugee process. I think that we should avoid mashing the two together because they are used in different ways.